model.py

```diff
@@ -202,9 +202,9 @@ def screen_shot(pixbuf):
 
     filename = 'fp%03d.png' % i
     filepath = os.path.join(tmpdir,filename)
-    pixbuf.save(filepath,'png')
+    pixbuf.savev(filepath,'png')
 
-    from sugar.datastore import datastore
+    from sugar3.datastore import datastore
```

Comment:
Shouldn't this import be at top of file?

Reply:
Thanks I will do